core/embed/rust/src/ui/model_mercury/layout.rs

```diff
@@ -1758,12 +1758,18 @@ pub static mp_module_trezorui2: Module = obj_module! {
     ///     *,
     ///     title: str | None,
     ///     address: str,
-    ///     amount: str,
+    ///     amount: str | None,
     ///     chunkify: bool,
     ///     account: str | None,
     ///     account_path: str | None,
     ///     br_code: ButtonRequestType,
     ///     br_name: str,
+    ///     summary_items: Iterable[tuple[str, str]] | None = None,
+    ///     fee_items: Iterable[tuple[str, str]] | None = None,
+    ///     summary_title: str | None = None,
+    ///     summary_br_code: ButtonRequestType | None = None,
+    ///     summary_br_name: str | None = None,
+    ///     cancel_text: str | None = None,
     /// ) -> LayoutObj[UiResult]:
     ///     """Confirm recipient."""
```

Comment:
This documentation "Confirm recipient." does not seem adequate anymore :smile:

Reply:
https://github.com/trezor/trezor-firmware/pull/4095/commits/ec6a2a8c8d7e840f3b19e2ab09c9bb08806c1449#diff-e939e2f37ae15d507723cbfa63ffa095f904d9e81ec403ddecb57c09ac09d3f4R1774